cmd/src/server.rs

```diff
@@ -186,12 +187,20 @@ fn run_raft_server(pd_client: RpcClient, cfg: &TiKvConfig, security_mgr: Arc<Sec
         None
     };
 
+    let mut gc_worker = GCWorker::new(
```

Comment:
`GCWorker` and `GCManager` can start together. It's not related to this PR. Maybe we can extract a single `start` method later.